packages/core/src/VDocument.ts

```diff
@@ -75,17 +78,18 @@ export class VDocument {
      * Get the format for the next insertion.
      */
     _getCurrentFormat(): FormatType {
+        const range = this.selection.range;
         let format: FormatType = {};
         if (this.formatCache) {
             return this.formatCache;
-        } else if (this.range.isCollapsed()) {
-            const charToCopyFormat = (this.range.start.previousSibling(isChar) ||
-                this.range.start.nextSibling(isChar) || {
+        } else if (range.isCollapsed()) {
+            const charToCopyFormat = (range.start.previousSibling(isChar) ||
+                range.start.nextSibling(isChar) || {
```

Comment:
Out of scope but this bit here is wrong and will have to disappear soon... Just wanted to pull your attention on it.